src/config/theme/components/layout.ts

```diff
@@ -20,39 +20,50 @@ const layout = {
       justifyContent: 'space-between',
       alignItems: 'center',
     },
+    title: {
+      color: 'toolbarPrimary',
+    },
+    subtitle: {
+      color: 'toolbarSecondary',
+    },
```

Comment:
Let's use the `text` color here. In case of the variant, you can use the `textSecondary` color.

Reply:
title and subtitle dark ?

Reply:
not following these https://github.com/reactcostarica/coronaviruscr/pull/16#discussion_r395877397  https://github.com/reactcostarica/coronaviruscr/pull/16#discussion_r395877716 https://github.com/reactcostarica/coronaviruscr/pull/16#discussion_r395878925